apps/mull-ui/src/generated/graphql.tsx

```diff
@@ -0,0 +1,612 @@
+import { gql } from '@apollo/client';
```

Comment:
For this generated file, it makes sense to not run checks on it. Here's how:
* For prettier, just add the file path to the `.prettierignore`:
```
...
apps/mull-ui/src/generated/graphql.tsx # add this line
```

* For pre-commit, modify line 8 of the `.pre-commit-config.yaml` file as such:
```
...
exclude: 'package-lock.json|apps/mull-ui/src/generated/graphql.tsx'
...
```